Jenkinsfile

```diff
@@ -76,6 +76,8 @@ pipeline {
               "^tests/packaging.*",
               "^vendor/github.com/elastic/beats.*"
             ]
+            // It requires to use /dev/null with tail to avoid the mage output installation
+            setEnvVar('BEAT_VERSION', sh(label: 'Get beat version', script: 'make get-version 2>/dev/null | tail -n 1', returnStdout: true)?.trim())
```

Comment:
I could not find another way to gather only the version, a similar approach is done for beats, the only difference is the mage installation that's required to happen when calling the get-version target goal, and therefore it's required to parse the output.

Reply:
You could make use of the version specified in https://github.com/graphaelli/apm-server/blob/master/cmd/version.go

Reply:
I was tempted to do so, but since the beats project already provides the `mage dumpVariables` goal, so I use the same approach as done for them -> https://github.com/elastic/beats/blob/d97fe08d6d5bb27e4c5e52d04a45448fd1fe8827/Makefile#L194-L197

Therefore if the versioning implementation changes then most likely the `dumpVariables` target should change as well, otherwise, we might implement two different versioning parsing implementations, that's the reason for this approach

Reply:
In fact, something like that was in place -> https://github.com/elastic/beats/pull/22027/commits/ea2f8148cd219c3673f189734307e8a44da711db#diff-5edb5ab93fc95960129eecb2ee3d8763a9c5e02b56e5b80126e0c5a52296ee8fL220 but I moved to the make goal

Do you see any issues with this approach?